.eslintrc.js

```diff
@@ -39,5 +48,7 @@ module.exports = {
         ignoreRestSiblings: false,
       },
     ],
+    '@typescript-eslint/no-var-requires': 'off',
+    '@typescript-eslint/no-empty-function': 'off',
```

Comment:
disabling these rules allow for the linter not to give any errors, leaving the codebase at the same linting configuration as previous